### DISCUSSION TITLE
MacOS 12.3 (Monterey) availability

### DISCUSSION THREAD

**Discussion by tumatauenga** [actions/virtual-environments#5283] [General questions] [answered]
Just wondering what the timeframe is on making the macOS 12 image available publicly? I saw the merge of the [new readme file](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-12-Readme.md) for it on 3/23,  so I assume that it's close, but just thought I'd ask if there is a specific date that I should check back in.  The fact that it's unavailable is causing a slight issue for me, because I am using Xcode 13.3 locally, and there are some changes to the `Package.resolved` file in the new version of Xcode that are incompatible with Xcode 13.2.1, which is the latest version supported in mac OS 11.

Thanks!

**Comment by miketimofeev** [comment, marked as answer]
Hi @tumatauenga!
We can't provide any ETA regarding macOS public availability but you can submit a request for a private preview by filling in [this form](https://docs.google.com/forms/d/e/1FAIpQLScBYuIzySEU1AXVkK32gVoi95_dEz2qLNqt0QMnaJgoKMZ5YA/viewform)

**Reply by gereons** [reply]
Related to this question: is there a way for those in the beta to check the deployment status of macos-12 images, 20220323.2 specifically? AFAICT this is the first version to include the release version of Xcode 13.3, which we need to be able to upload apps to Apple.

**Reply by miketimofeev** [reply]
@gereons you can follow the releases page https://github.com/actions/virtual-environments/releases (pre-release means that it's currently being deployed)

**Reply by tonipepperoniuk** [reply]
@miketimofeev Is there any way to sign up for private preview in Azure DevOps? Thanks.

**Reply by miketimofeev** [reply]
@tonipepperoniuk I'm afraid it's not yet available but we will announce it as soon as it is ready.

**Reply by eschwieb** [reply]
@miketimofeev Microsoft Office (yes, us) really needs this ASAP in ADO.

**Comment by tumatauenga** [comment]
I'm a little confused over whether this is actually available yet. On the releases page there appears to be releases of it without the "pre-release" flag now, however I still don't see it listed on the [home page](https://github.com/actions/virtual-environments) as an available environment?

**Reply by miketimofeev** [reply]
@tumatauenga pre-release means that the image is being rolled out at the moment so it's unrelated to image GA.

**Reply by tumatauenga** [reply]
Oh OK, so it is still in pre-release then? Because releases like [this one](https://github.com/actions/virtual-environments/releases/tag/macOS-12%2F20220412.1) don't appear to have the pre-release tag, which is where I'm getting confused. Could you please let me know exactly what I should be watching to know that it's ready and available to use (specifically within Azure DevOps).  Should I just be refreshing the [home page](https://github.com/actions/virtual-environments)?

**Reply by miketimofeev** [reply]
it is in private preview, yes. You can submit one of the forms if you'd like to get access
- Azure DevOps https://forms.gle/3DzreCjz4V2DR8aB7
- GitHub actions https://forms.gle/D1ihWjw3xDfjYbu67

**Comment by DanielKrofchick** [comment]
Looks like it's in public beta [now](https://github.blog/changelog/2022-04-25-github-actions-public-beta-of-macos-12-for-github-hosted-runners-is-now-available/).

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
